components/ImageBox.tsx

```diff
@@ -0,0 +1,15 @@
+import Image from 'next/image'
+
+type Props = {
+  src: StaticImageData
+  width: number
+  alt?: string
+}
```

Comment:
```suggestion
type Props = React.ComponentProps<typeof Image>
```

This will extract all props that available in `Image` component to our `Props` type